### PR TITLE
#170708806 view all parkings endpoint

### PR DESCRIPTION
**Pull request by Elvis-rugamba**
### What does this PR do?
Have view all parkings endpoint
### Description of Task to be completed?
Having `GET/api/parkings` endpoint working
### How should this be manually tested?
After cloning the repo, cd into it and run the following commands:

- npm install

- npm run dev

Using Postman test endpoint above
### What are the relevant pivotal tracker stories?
#170708806

### CODE REVIEW

**Review by niyodusengaclement — COMMENTED** [review]
Looks well but you changed the test script

**Comment by istafrica-dennw** [comment]
hey reviewers. please review this PR

File: package.json [hunk 1]
```diff
@@ -6,8 +6,13 @@
   "scripts": {
     "start": "cross-env NODE_ENV=production npm run createTables && cross-env NODE_ENV=production babel-node ./src/server.js",
     "dev": "cross-env NODE_ENV=production node_modules/.bin/nodemon ./src/server.js --exec babel-node --",
```
Comment (Skemc) [inline package.json:8]: we need to first resolve this conflict

**Review by Skemc — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -6,8 +6,13 @@
...
+<<<<<<< HEAD
     "test": "mocha  --timeout 5000 --require @babel/polyfill --require @babel/register --exit ./test/index.test.js",
     "createTables": "node --require @babel/polyfill --require @babel/register src/db/createTables.js",
+=======
+    "test": "echo \"Error: no test specified\" && exit 1",
```
Comment (shaazk) [inline package.json:9]: there is a conflict in the test script, did you edit it or? please resolve it

**Pull request merged**